src/main/java/com/meetup/server/user/domain/User.java

```diff
@@ -59,4 +69,25 @@ public void updateAgreement(boolean personalInfoAgreement, boolean marketingAgre
     public void updateNickname(String nickname) {
         this.nickname = nickname;
     }
+
+    public void updateToWithdraw() {
+        this.profileImage = null;
+        this.personalInfoAgreement = false;
+        this.marketingAgreement = false;
+        this.nickname = WITHDRAWN_NICKNAME;
+        this.role = Role.WITHDRAWN;
+        this.deletedAt = LocalDateTime.now();
+        this.isDeleted = true;
+    }
+
+    private static final String WITHDRAWN_NICKNAME = "알 수 없음";
+
+    public void updateUser(User user) {
```

Comment:
P3) 위 리뷰와 마찬가지로 유저의 재가입 행위를 드러내면 좋을 거 같습니다!

```suggestion
    public void rejoin(User user) {
```